src/common/utils/create_entity.py

```diff
@@ -69,13 +70,22 @@ def is_json(attr: BlueprintAttribute):
             ""
         ]  # type(attr.default) == dict or type(attr.default) == list
 
-    # add all non optional attributes with default value.
     # type is inserted based on the parent attributes type, or the initial type for root entity.
     def _get_entity(self, blueprint: Blueprint, entity: dict):
+        """
```

Comment:
Great! Could this be moved up to line 26? As a docstring to the class instead of a private function?